### PR TITLE
Serve CORS headers

### PR DESCRIPTION
**Pull request by makyo**

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/bundleservice/17/
Test PASSed.

**Comment by makyo** [comment]
QA: `curl -Sv -X OPTIONS localhost:8000/bundlechanges/fromYAML` should show cors headers

File: server.go [hunk 1]
```diff
@@ -17,17 +17,51 @@ import (
 	"github.com/juju/bundleservice/params"
 )
 
+// server represents a bundleservice HTTP server
+type server struct {
+	router *httprouter.Router
+}
+
+// ServeHTTP implements http.Handler.Handle.
```
Comment (frankban) [inline server.go:25]: Just http.Handler?

**Review by jrwren — APPROVED** [review]
👍

**Review by frankban — APPROVED** [review]
Thanks!

**Comment by jrwren** [comment]
QA exactly as stated

```
$ curl -Sv -X OPTIONS localhost:8000/bundlechanges/fromYAML
*   Trying ::1...
* Connected to localhost (::1) port 8000 (#0)
> OPTIONS /bundlechanges/fromYAML HTTP/1.1
> Host: localhost:8000
> User-Agent: curl/7.43.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Access-Control-Allow-Credentials: true
< Access-Control-Allow-Headers: Bakery-Protocol-Version, Macaroons, X-Requested-With, Content-Type
< Access-Control-Allow-Methods: POST,OPTIONS
< Access-Control-Allow-Origin: *
< Access-Control-Cache-Max-Age: 600
< Access-Control-Expose-Headers: WWW-Authenticate
< Date: Mon, 19 Sep 2016 17:11:01 GMT
< Content-Length: 0
< Content-Type: text/plain; charset=utf-8
<
* Connection #0 to host localhost left intact
```

**Comment by makyo** [comment]
Thanks all :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/bundleservice-merge

**Pull request merged**